src/pages/main-page/MainPage.jsx

```diff
@@ -0,0 +1,34 @@
+import { useDispatch, useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+
+import News from '../../components/news-card/News';
+import Alert from '../../components/alert/Alert';
+import { getNewsRequest } from '../../redux/actions/news';
+import Loader from '../../components/loader/Loader';
+
+import './MainPage.css';
+
+function MainPage() {
+  const { news, isLoading, error } = useSelector((state) => state.news);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getNewsRequest());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+  if (error) {
+    return <Alert message={error} severity="error" />;
+  }
+  if (!news.length) {
+    return <Alert message="News not found" severity="info" />;
+  }
+  return (
+    <div className="main-page">
+      {news.map((post) => <News post={post} />)}
+    </div>
+  );
+}
+export default MainPage;
```

Comment:
The page is also a component, move the main-page folder to components and rename it